src/main/default/lwc/descriptionInput/descriptionInput.html

```diff
@@ -0,0 +1,16 @@
+<template>
+  <c-section-title title={label}></c-section-title>
+  <div class={formClass}>
+    <lightning-input-rich-text
+      formats={formats}
+      value={value}
+      onchange={handleChange}
+      aria-describedby="description-error-message"
+    ></lightning-input-rich-text>
+    <template if:true={hasError}>
```

Comment:
Instead of displaying the error message yourself, you should leverage the validation functionalities offered by the `lightning-input-rich-text` component. See the "Input Validation" section of [this page](https://developer.salesforce.com/docs/component-library/bundle/lightning-input-rich-text/documentation).

Reply:
Thanks! I updated this 👌